### ISSUE TITLE
Build Failure

### ISSUE THREAD

**Issue by felipetesc** [haxeui/raylib-haxe#1]
## Expected Behavior

To build the example contained inside the readme

## Current Behavior

 Build Failure

## Possible Solution

I don't know how to solve this, but perhaps it's related to how Haxe links the source to Windows, and by reading the output it seems a issue related to a missing file: Build.xml

## Steps to Reproduce (for bugs)

1. Open Windows Powershell
2. Type 'haxe build.hxml
3. Wait
4. Read the message

```bat
C:\Users\MyUser\Desktop\HashLink\raylib_sample>haxe build.hxml
haxelib run hxcpp Build.xml haxe -Dhaxe="4.2.5" -Dhaxe3="1" -Dhaxe4="1" -Dhaxe_ver="4.205" -Dhxcpp="4.2.1" -Dhxcpp_api_level="400" -Dhxcpp_smart_strings="1" -Draylib-haxe="1.1.0" -Draylib_haxe="1.1.0" -Dsource_header="Generated by Haxe 4.2.5" -Dstatic="1" -Dtarget.name="cpp" -Dtarget.static="true" -Dtarget.sys="true" -Dtarget.threaded="true" -Dtarget.unicode="true" -Dtarget.utf16="true" -Dutf16="1" -I"src/" -I"C:\\HaxeToolkit\\haxe\\lib\\hxcpp/4,2,1/" -I"C:\\HaxeToolkit\\haxe\\lib\\raylib-haxe/1,1,0/" -I"" -I"C:\\HaxeToolkit\\haxe\\extraLibs/" -I"C:\\HaxeToolkit\\haxe\\std/cpp/_std/" -I"C:\\HaxeToolkit\\haxe\\std/"
Error: Could not find include file "C:/HaxeToolkit/haxe/lib/raylib-haxe/1,1,0/Build.xml"
Error: Build failed
```
## Media

Not aplied

## Test app / minimal test case

I've just followed the example from README.md

## Context

Can't even test the lib

## Your Environment

* Version used: 1,1,0
* Environment name and version : No
* Operating System and version (desktop or mobile): Windows 11 
* Link to your project: No

**Comment by felipetesc** [comment]
To fix: 
I have added Build.xml inside raylib-haxe\1,1,0\lib\raylib\
For some reason haxelib command  is not installing several files needed to build the project

**Comment by ianharrigan** [comment]
Hmmm, ill check the haxelib install, maybe the zip was bad

**Comment by felipetesc** [comment]
> Hmmm, ill check the haxelib install, maybe the zip was bad

I fixed all issues after deleting raylib inside C:\HaxeToolkit\haxe\lib\raylib-haxe\1,1,0\ and copying the folder from github raylib-haxe/tree/main/lib/raylib inside C:\HaxeToolkit\haxe\lib\raylib-haxe\1,1,0\

**Comment by ianharrigan** [comment]
Cool - so sounds like a bad haxelib release on my part

**Comment by Its-Kenta** [comment]
Also facing this issue but the solution above did not fix it for me.

**Comment by ianharrigan** [comment]
Have you tried git versions of haxeui-core and haxeui-raylib?

**Comment by Its-Kenta** [comment]
I didnt know you need to use haxeui-core? Is there any reason behind that?

**Comment by ianharrigan** [comment]
whoops, sorry, not reading correctly AT ALL... i thought this was the haxeui-raylib repo, ntit the raylib-haxe repo. So no, you shouldnt need haxeui at all. So, have you tried using the git version of this lib?

**Comment by Its-Kenta** [comment]
My bad as well! I finally realised what was needed.
I did so, but I ran into few problems in terms of using it so I shamefully gave up.. Sorry!

**Comment by ianharrigan** [comment]
problems compiling? Or problems using?

**Comment by Its-Kenta** [comment]
Using really, maybe it's my lack of knowledge in Haxe as of yet. Not sure.

**Comment by ianharrigan** [comment]
Right, well, feel free to post any problems you are having here, i might be able to help (or not) :) 

Cheers,
Ian
